src/Rules/StarterWeb/AI/IndividualAuth/project.json

```diff
@@ -29,7 +29,7 @@
   },
 
   "commands": {
-    "web": "Microsoft.AspNet.Server.Kestrel",
+    "web": "$safeprojectname$",
```

Comment:
The reason why we removed multiple commands in our samples is to prepare for the move towards dotnet cli where commands don't exist. Maybe we should do the same here and remove `ef`.